json_serializable/test/json_serializable_test.dart

```diff
@@ -512,14 +510,12 @@ abstract class _$SubTypeSerializerMixin {
 
 final _formatter = new dart_style.DartFormatter();
 
-Future<CompilationUnit> _getCompilationUnitForString(String projectPath) async {
-  var fileName = 'json_serializable_test_input.dart';
-  var filePath = p.join(getPackagePath(), 'test', 'src', fileName);
-  var source =
-      new StringSource(new File(filePath).readAsStringSync(), fileName);
-
-  var context = await getAnalysisContextForProjectPath(projectPath);
+Future<CompilationUnit> _getCompilationUnitForString() async {
```

Comment:
`ForString`... what String?

Reply:
Old name. Can fix it up in a follow-up CL